service/src/main/java/bio/terra/pipelines/stairway/steps/common/QuotaConsumedValidationStep.java

```diff
@@ -63,16 +67,18 @@ public StepResult doStep(FlightContext flightContext) {
     Map<?, ?> quotaOutputsMap = workingMap.get(ImputationJobMapKeys.QUOTA_OUTPUTS, Map.class);
     Object quotaConsumedObj = quotaOutputsMap.get("quotaConsumed");
     if (quotaConsumedObj == null) {
+      logger.error("Missing 'quotaConsumed' entry in quota outputs map.");
       return new StepResult(
-          StepStatus.STEP_RESULT_FAILURE_FATAL,
-          new InternalServerErrorException("Missing 'quotaConsumed' entry in quota outputs map."));
+          StepStatus.STEP_RESULT_FAILURE_FATAL, new PipelineInternalServerException());
     }
     String rawQuotaConsumedValue = quotaConsumedObj.toString();
     int rawQuotaConsumed = Integer.parseInt(rawQuotaConsumedValue);
     if (rawQuotaConsumed <= 0) {
+      logger.error(
+          "Quota consumed is unexpectedly not greater than 0. Quota consumed value: {}",
+          rawQuotaConsumed);
       return new StepResult(
-          StepStatus.STEP_RESULT_FAILURE_FATAL,
-          new InternalServerErrorException("Quota consumed is unexpectedly not greater than 0"));
+          StepStatus.STEP_RESULT_FAILURE_FATAL, new PipelineInternalServerException());
```

Comment:
would we rather the message returned say something about quota being an issue?  I think there is an area between one generic message for all exception and more helpful non internal messages we can return.  Curious what your and other's thoughts are on it.